src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.ts

```diff
@@ -27,15 +27,11 @@ import { AdditionalServicesUtils } from '../additional-services-utils.service';
 })
 export class AdditionalServiceModalComponent {
   category: any;
-  isServiceAvailable = false;
-  isServiceEnabled = false;
-  serviceProcessName = '';
-  serviceType = '';
   serviceName = '';
-  isInstalled: boolean;
-  packageName = '';
+  enabled: Boolean;
   btnText = 'Add';
   showDeleteBtn = true;
+  isServiceEnabled = false;
```

Comment:
We need to keep `enabled` and `isServiceEnabled`? If Yes keep these together with an inline comment and how these differs to  serviceInfo.isEnabled?